### PR TITLE
Move custom templates over to v9 storybook

### PR DESCRIPTION
**Pull request by micahgodbolt**
Move custom templates over to v9 storybook site and enable themes and docs build.

### CODE REVIEW

File: apps/public-docsite-v9/.storybook/main.js [hunk 1]
```diff
@@ -3,17 +3,7 @@ const rootMain = require('../../../.storybook/main');
...
 module.exports = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(ts|tsx)', ...utils.getVnextStories()],
-  addons: [
-    {
-      name: '@storybook/addon-docs',
-      options: {
-        configureJSX: true,
-        babelOptions: {},
-        sourceLoaderOptions: null,
-        transcludeMarkdown: true,
```
Comment (micahgodbolt) [inline apps/public-docsite-v9/.storybook/main.js:13]: This is currently throwing an error when enabled. I couldn't see a difference between when using default options vs these custom options. Will log followup

File: apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx [hunk 2]
```diff
@@ -35,19 +34,12 @@ const useStyles = makeStyles({
   container: {
     flexGrow: 1,
   },
-  // style overrides for when hosted in website
```
Comment (micahgodbolt) [inline apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx:38]: hosted styles no longer needed as we aren't going through with iframe embedding

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `178.614 kB`<br />`49.862 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.671 kB`<br />`10.696 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/eab6f0f033b2fad90a25c421ab19f85d98417743'>eab6f0f033b2fad90a25c421ab19f85d98417743</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/Avatar-mount_base.html">1071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/Avatar-mount.html">1076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/Button-mount_base.html">649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/Button-mount.html">649</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProvider-mount_base.html">2137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProvider-mount.html">2180</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-mount_base.html">339</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-mount.html">333</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">273</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">288</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">350</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/MakeStyles-mount_base.html">1872</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22599/perf-test-react-components/MakeStyles-mount.html">1842</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (layershifter) [inline change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json:2]:
```suggestion
  "type": "none",
```

? 🤔

Comment (micahgodbolt) [reply change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json]: yup, accidentally hit the default value for type

File: apps/public-docsite-v9/src/DocsComponents/ThemePicker.stories.tsx [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
+import {
+  MenuButton,
+  Menu,
+  MenuPopover,
+  MenuItemRadio,
+  MenuList,
+  MenuTrigger,
+  makeStyles,
+  MenuProps,
+} from '@fluentui/react-components';
```
Comment (khmakoto) [inline apps/public-docsite-v9/src/DocsComponents/ThemePicker.stories.tsx:11]: _nit:_ Order these alphabetically

```suggestion
import {
  makeStyles,
  Menu,
  MenuButton,
  MenuItemRadio,
  MenuList,
  MenuPopover,
  MenuProps,
  MenuTrigger,
} from '@fluentui/react-components';
```

File: change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Move custom templates over to v9 storybook",
```
Comment (khmakoto) [inline change/@fluentui-react-components-3f01dab2-bda7-4bbf-959b-f0eb66365bfc.json:3]:
```suggestion
  "comment": "Move custom templates over to v9 storybook.",
```

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**